static/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -913,3 +913,41 @@ const ERROR_LEVEL_WEIGHTS: Record<TraceError['level'], number> = {
   sample: 2,
   info: 1,
 };
+
+/**
+ * Formats start and end unix timestamps by inserting a leading zero if needed, so they can have the same length
+ */
+export function getFormattedTimeRangeWithLeadingZero(start: number, end: number) {
```

Comment:
I was not sure if this function should be here or somewhere in the common `utils` folder